website/docs/terms/etl.md

```diff
@@ -0,0 +1,124 @@
+---
+id: etl
+title: ETL
+displayText: ETL
+hoverSnippet: Extract, Transform, Load (ETL)is the process of first extracting data from a data source, transforming it, and then loading it into a target data warehouse.
+---
+
+ETL, or “Extract, Transform, Load”, is the process of first extracting data from a data source, transforming it, and then loading it into a target data platform. ETL is contrasted with the newer <Term id="elt">Extract, Load, Transform (ELT)</Term> workflow, where transformation occurs after data has been loaded into the target data warehouse.
+
+The ETL workflow is still an effective method to create pipelines to bring data from different sources to end business users. However, the past decade has brought an immense amount of innovation in this sphere. There are now a variety of low or no-code data extractors, a code-based and version controlled transformation layer, and several affordable cloud storage options that have allowed ELT to become a competitive analytics workflow. 
+
+It’s important to talk about ETL and understand how it works, where it provides value, and how it can hold people back. If you don’t talk about the benefits and drawbacks of systems, how can you expect to improve them?
+
+## How ETL works
+
+In an ETL process, data is first extracted from a source, transformed, and then loaded into a target data platform. We’ll go into greater depth for all three steps below.
+
+![A diagram depicting the ETL workflow. The diagram starts by depicting raw data being extracted from various example data sources like an email CRM, Facebook Ads platform, a backend database, and Netsuite. Once the data is extracted, the raw data is transformed within the data pipeline via renaming, casting, joining, and enriching. After the data is transformed within the data pipeline, the modeled data is loaded into a data warehouse.](/img/docs/terms/etl/etl-diagram.png)
+
+### Extract
+
+In this first step, data is extracted from different data sources. Data that is extracted at this stage is likely going to be eventually used by end business users to make decisions. Some examples of these data sources include:
+
+- Ad platforms (Facebook Ads, Google Ads, etc.) 
+- Backend application databases 
+- Sales CRMs
+- And more!
+
+To actually get this data, data engineers may write custom scripts that make Application Programming Interface (API) calls to extract all the relevant data. Because making and automating these API calls gets harder as data sources and data volume grows, this method of extraction often requires strong technical skills. In addition, these extraction scripts also involve considerable maintenance since APIs change relatively often. Data engineers are often incredibly competent at using different programming languages such as Python and Java. Data teams can also extract from these data sources with open source and Software as a Service products.
```

Comment:
let's add (SaaS) after we write out 'Software as a Service' in the last line

Reply:
Fantastic catch!!!